waiter/resources/web/authenticated-redirect.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
+<head>
+    <title>Working...</title>
+</head>
+<body>
+<form action="<%= auth-redirect-uri %>" method="post">
+    <input type="hidden" name="saml-auth-data" value="<%= saml-auth-data %>">
+    <noscript>
+        <p>JavaScript is disabled. Click Continue to continue to your application.</p>
+        <input type="submit" value="Continue">
```

Comment:
We should make the html well-formed. We can close the input `/>`

Reply:
done